pkg/kritis/cryptolib/verifier.go

```diff
@@ -16,7 +16,18 @@ limitations under the License.
 
 package cryptolib
 
-import "errors"
+import (
+	"errors"
+	"fmt"
+)
+
+// TODO: Remove function stubs once implemented
```

Comment:
Always have a name or bug attached to TODOs

Reply:
Would you suggest using github issue as the bug system?

Reply:
That would work. I'd probably just use my ldap here because it's a little easier.

Reply:
ah, you mean assignee/reporter's name, that totally makes sense!

Reply:
Removed. Functions are no longer being globally stubbed.